### ISSUE TITLE
Epic: Dashboard

### ISSUE THREAD

**Issue by ri0t** [Hackerfleet/hfos#48]
* [x] Integrate gridster.js (via deckster.js :)
* [x] Add basic digital readout
* [ ] ~~Add basic analog readout~~
* [ ] ~~Add basic course readout~~
* [ ] ~~Add Misc~~
* [x] Store configuration
* [x] Read configuration
* [x] Update mechanism
* [ ] ~~Interactive adding/removing~~
* [x] Better css (hide unneeded elements)

**Comment by ij0n** [comment]
Ideas for Features:

"History" as a Graph in the Background of any digital or analoge Value. (adjustable timeframe)

little Graphic "Distance from Track" with Course to waypoint and the orthogonal distance from the planned track

Timers! (till end of watch, ETA to next waypoint, ETA to destination, UTC, local time, wake me in 10min)

distance sailed in last 24h
and other performance indicators

custom and dynamic limits, hard and soft, for EVERY value with alarms

alarm-box or maybe an overlay?

wind-display! measured (apparent) wind, true wind...

pitch, roll, yaw...history, current angles, 30min max...

**Comment by ri0t** [comment]
Blocked by #92

**Comment by ri0t** [comment]
* [ ] Time series visualization: http://square.github.io/cubism/
* [ ] Polar plotter: http://bl.ocks.org/jeffthink/1630683
* [ ] Quantitative time series: http://prcweb.co.uk/lab/circularheat/
* [ ] nice dials: http://bl.ocks.org/rmarimon/1144047

**Comment by ri0t** [comment]
The GOE takes care of the configuration. The default form is ugly as hell but can be customized :)

**Comment by ri0t** [comment]
Some stuff moved to another ticket, this one is to make sure, the dashboard works. It does now :cocktail:
